user-docs/transaction-functions.md

```diff
@@ -0,0 +1,121 @@
+# Transaction Functions available in `algob`
+
+## `getSuggestedParams`
+
+This function performs the task of fetching the common transaction params from the connected network blockchain. Private chains might throw some error while executing this function, please make sure that the node progresses in that case. 
```

Comment:
how about `This function returns suggested params`.?

Reply:
It's mentioned in the end. L11

Reply:
just that fetching common transactions doesn't seem correct

Reply:
I like Amit proposal. Try to make shorter sentences. Instead: "This function performs the task of fetching.." (we don't need to be that verbose) -> "Returns blockchain transaction suggested parameters".
Also, as noted in Discord - this is an API documentation, and should be in a doc strings. Here we should do more guide style description / tutorial.